WEB/controllers/tweets.py

```diff
@@ -28,9 +28,23 @@ def load_tweets():  # pylint: disable=unused-variable
         description: Server error occurred
     '''
     tweets = tweets_service.get_tweets_from_twitter()
+    # Parses raw JSON to filter out unneeded information.
+    data = []
+    for tweet in tweets["statuses"]:
+      parsed_tweet = {
+        "created_at": tweet["created_at"],
+        "id": tweet["id"],
+        "text": tweet["text"],
+      #  "entities": tweet["entities"],
+      #  "user": tweet["user"],
+        "geo": tweet["geo"],
+        "coordinates": tweet["coordinates"],
+        "place": tweet["place"]
+      }
+      data.append(parsed_tweet)
```

Comment:
Nice to have change: pull this out into its own method with a signature such as `(tweets) => data`, this will allow us to move this transformation around in the code easily if we need to and will also help a little with the second point which is:
Absolutely required: We should avoid using straight key accessors on dictionaries in python. Calling `tweet["place"]` will crash our server if "place" doesn't exist on the object. This is especially important as Twitter is an outside source of data and we want to make sure we don't open ourselves up to a bug on our side if they break contract (unlikely in this specific case but still important to follow with outside data). get accessors are much safer ways of accessing data.